src/RuleSet/Sets/PhpCsFixerSet.php

```diff
@@ -58,6 +58,7 @@ public function getRules(): array
             'fully_qualified_strict_types' => [
                 'import_symbols' => true,
             ],
+            'heredoc_closing_marker' => true,
```

Comment:
> if this fixer should be part of @PhpCsFixer

i wonder if we have any community standardising it (eg PER-CS), and we could re-use this. (also in context of what variant to use)

I'm OK to have it added to local, repo-based config file,
for us promoting it globally as best recommendation - let's gather more opinions.